README.md

```diff
@@ -53,4 +53,11 @@ docker run --publish 8080:8080 --mount type=bind,source="$(pwd)",destination=/ap
 cd /app/
 npm install
 npm start
-```
\ No newline at end of file
+```
+
+```
+docker run --mount type=bind,source="$(pwd)",destination=/app --interactive --tty node:latest /bin/bash
```

Comment:
This might be better represented as a docker-compose.yml spec:

- `docker-compose up web`
- `docker-compose up test`
- `docker-compose up lint`